packages/cadl-ranch-specs/http/azure/resource-manager/models/resources/mockapi.ts

```diff
@@ -17,9 +17,9 @@ const validTopLevelResource = {
   systemData: {
     createdBy: "AzureSDK",
     createdByType: "User",
-    createdAt: new Date(),
+    createdAt: "2024-10-04T00:56:07.442Z",
```

Comment:
Design Change 1: The `new Date()` is changed to a fixed date. This enables the validation to be successful. Else, the server and client time will always be different and will cause error.